account.go

```diff
@@ -59,7 +59,7 @@ type AccountParams struct {
 	ExternalAccount           *AccountExternalAccountParams
 	LegalEntity               *LegalEntity
 	TransferSchedule          *TransferScheduleParams
-	Managed, DebitNegativeBal bool
+	Managed, DebitNegativeBal *bool
```

Comment:
Changing the type of the variable is a breaking change and not something I'd recommend. You'd need to follow the solutions we've used in other PRs such as https://github.com/stripe/stripe-go/pull/275 and use another parameter for this. It's not pretty but ensures that it doesn't break someone's integration